modules/library/src/test/scala/zio/elasticsearch/ElasticQuerySpec.scala

```diff
@@ -2531,6 +2531,50 @@ object ElasticQuerySpec extends ZIOSpecDefault {
           assert(queryWithScoreMode.toJson(fieldPath = None))(equalTo(expectedWithScoreMode.toJson)) &&
           assert(queryWithAllParams.toJson(fieldPath = None))(equalTo(expectedWithAllParams.toJson))
         },
+        test("prefix") {
+          val query                    = prefix(TestDocument.stringField, "test")
+          val queryWithCaseInsensitive = prefix(TestDocument.stringField, "test").caseInsensitiveTrue
+          val queryWithAllParams       = prefix(TestDocument.stringField, "test").caseInsensitiveFalse
```

Comment:
Remove this check.

Reply:
We have only one parameter, so `queryWithAllParams` is same as `queryWithCaseInsensitive`.